apps/web/src/lib/components/ui/agent-editor/from-create.svelte

```diff
@@ -36,11 +37,11 @@
 		</Dialog.Trigger>
 	</div>
 	<Dialog.Content class="w-full sm:max-w-full lg:max-w-4xl">
-		<Dialog.Header>
+		<!-- <Dialog.Header>
 			<Dialog.Title>Create Agent</Dialog.Title>
-		</Dialog.Header>
+		</Dialog.Header> -->
```

Comment:
nit: avoid commented out code, it'll remain in git history so you can get it back later